README.md

```diff
@@ -289,6 +290,30 @@ require('ascoltatori').build({ json: false }, function(err, a) {
 });
 ```
 
+### Apache Kafka
+
+```javascript
+var ascoltatori = require('ascoltatori');
+var settings = {
+  type: 'kafka',
+  json: false,
+  kafka: require("kafka-node"),
+  connectString: "localhost:2181",
+  clientId: "ascoltatori",
+  groupId: "ascoltatori",
+  topics: [
+    "hello",
+    "hello_42_there",
+    {topic: "hello_42", encoding: "utf8"},
+    {topic: "image", encoding: "buffer"}
+  ]
```

Comment:
I think this is the kind of behavior that should be handled inside Ascoltatori. Topics should be created automatically by the library, rather than predefined in the configuration.